<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Library_MainWindow</class>
 <widget class="QMainWindow" name="Library_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library_S17</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Search_Box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Search_Button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Show_Table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>661</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="test_my">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>104</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
